stable/bitcoind/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+name: bitcoind
+version: 0.1.0
+appVersion: "0.15.1.0"
+description: Bitcoin is an innovative payment network and a new kind of money.
+keywords:
+  - bitcoind
+  - cryptocurrency
+  - blockchain
+home: https://bitcoin.org/
+icon: https://bitcoin.org/img/icons/logotop.svg
+sources:
+  - https://github.com/kubernetes/charts
+  - https://github.com/kuberstack/bitcoind
+  - https://github.com/kylemanna/docker-bitcoind
+maintainers:
+  - name: daniel-yavorovich
+    email: daniel@arilot.com
```

Comment:
You don't want to add yourself as maintainer?

Reply:
Just forgot.